src/collection/components/AddFragment.tsx

```diff
@@ -4,34 +4,54 @@ import React, { FunctionComponent } from 'react';
 import { Button, Container, ToolbarItem } from '@viaa/avo2-components';
 import { Avo } from '@viaa/avo2-types';
 
+import { ContentBlockType } from '../types';
+
+const COLLECTION_CONTENT_BLOCKS = ['RichText'];
```

Comment:
Deze eventueel ook in de constants file zetten?
Also check comment op: `src/collection/types.ts`